.github/workflows/build_and_publish.yml

```
@@ -8,7 +8,7 @@ on:
 jobs:
  build-n-publish:
   name: Build and publish Python distribution to PyPI
-  runs-on: ubuntu-18.04
+  runs-on: ubuntu-latest
   steps:
    - name: Check out git repository
      uses: actions/checkout@v2
```

And some version update 🆙 

```suggestion
     uses: actions/checkout@v3
     
      - name: Set up Python 3.7
     uses: actions/setup-python@v4
     
      - name: Publish distribution 📦 to PyPI
        uses: pypa/gh-action-pypi-publish@release/v1
```